packages/squiggle-lang/__tests__/Distributions/Invariants/Means_test.res

```diff
@@ -0,0 +1,123 @@
+open Jest
```

Comment:
I'd probably write a few sentences on the top of this describing the strategy here. Like, what the tests are really testing, and how they are organized.